integration-tests/src/test/groovy/com/okta/sdk/tests/it/AuthorizationServerIT.groovy

```diff
@@ -71,6 +61,14 @@ class AuthorizationServerIT extends ITSupport {
         assertThat(createdAuthorizationServer.getAudiences(), contains("api://example"))
     }
 
+    @Test
```

Comment:
This test ideally must create Authorization server (within the same test's scope) before listing it. This test is unnecessary as the test `listCreatedAuthorizationServerTest()` tests the same.

Reply:
fixed

Reply:
You should `createAuthorizationServer()` before you list `listAuthorizationServers()`.